### PR TITLE
add lighting talk videos

### PR DESCRIPTION
**Pull request by daviddias**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,30 +98,33 @@ You can find the content description, speakers and trainers at:
...
+| <img src="https://avatars3.githubusercontent.com/u/11004800?s=460&v=4" width="100" /> | Jonathan Budd ([@cesarosum](https://github.com/cesarosum)) | Peergos: E2E encrypted storage and social network on IPFS | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-peergos.pdf) | [Video](https://youtu.be/MBHsJlx7knA) |
+| <img src="https://avatars3.githubusercontent.com/u/1017236?s=460&v=4" width="100" /> | André Cruz ([@satazor](https://github.com/satazor)) | Introducing Nomios | [Website](https://nomios.io/) | [Video](https://youtu.be/EEIq6Eu9ug4) |
+| <img src="https://avatars0.githubusercontent.com/u/17089485?s=460&v=4" width="100" /> | Alexandre Trottier ([@postables](https://github.com/postables)) | Why open source matters and how individuals can further the movement | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-opensource.pdf) | [Video](https://youtu.be/usa796ebEnI) |
+| <img src="https://avatars3.githubusercontent.com/u/370259?s=460&v=4" width="100" /> | Andrew Hill ([@andrewxhill](https://github.com/andrewxhill)) | Textile | SLIDES | [Video](https://youtu.be/uI9eEZQZpfw)
```
Comment (daviddias) [inline README.md:125]: @andrewxhill do you happen to have an url to the slides of your Textile Lighting Talk?

File: README.md [hunk 2]
```diff
@@ -98,30 +98,33 @@ You can find the content description, speakers and trainers at:
...
+| | Matt Ober ([@obo20](https://github.com/obo20)) | IPFS Industry Trends | SLIDES | [Video](https://youtu.be/LumqUfWd6g0)
```
Comment (daviddias) [inline README.md]:
```suggestion
| <img src="https://avatars2.githubusercontent.com/u/7811297" width="100" /> | Matt Ober ([@obo20](https://github.com/obo20)) | IPFS Industry Trends | SLIDES | [Video](https://youtu.be/LumqUfWd6g0)
```

File: README.md [hunk 3]
```diff
@@ -98,30 +98,33 @@ You can find the content description, speakers and trainers at:
...
+| <img src="https://avatars3.githubusercontent.com/u/1017236?s=460&v=4" width="100" /> | André Cruz ([@satazor](https://github.com/satazor)) | Introducing Nomios | [Website](https://nomios.io/) | [Video](https://youtu.be/EEIq6Eu9ug4) |
+| <img src="https://avatars0.githubusercontent.com/u/17089485?s=460&v=4" width="100" /> | Alexandre Trottier ([@postables](https://github.com/postables)) | Why open source matters and how individuals can further the movement | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-opensource.pdf) | [Video](https://youtu.be/usa796ebEnI) |
+| <img src="https://avatars3.githubusercontent.com/u/370259?s=460&v=4" width="100" /> | Andrew Hill ([@andrewxhill](https://github.com/andrewxhill)) | Textile | SLIDES | [Video](https://youtu.be/uI9eEZQZpfw)
+| <img src="https://avatars2.githubusercontent.com/u/7811297" width="100" /> | Matt Ober ([@obo20](https://github.com/obo20)) | IPFS Industry Trends | SLIDES | [Video](https://youtu.be/LumqUfWd6g0)
```
Comment (daviddias) [inline README.md:126]: @obo20 do you happen to have an url to the slides of your IPFS Industry Trends lighting talk?

File: README.md [hunk 4]
```diff
@@ -98,30 +98,33 @@ You can find the content description, speakers and trainers at:
...
+| | Zhanna Sharipova ([]()) | Anytype | SLIDES | [Video](https://youtu.be/NAF-4AFSqIo)
```
Comment (daviddias) [inline README.md]:
```suggestion
| <img src="https://avatars1.githubusercontent.com/u/687840" width="100" /> | Zhanna Sharipova ([@sharipova](https://github.com/sharipova)) | Anytype | SLIDES | [Video](https://youtu.be/NAF-4AFSqIo)
```

File: README.md [hunk 5]
```diff
@@ -98,30 +98,33 @@ You can find the content description, speakers and trainers at:
...
+| <img src="https://avatars0.githubusercontent.com/u/17089485?s=460&v=4" width="100" /> | Alexandre Trottier ([@postables](https://github.com/postables)) | Why open source matters and how individuals can further the movement | [PDF](https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/ipfscamp2019-lightningtalk-opensource.pdf) | [Video](https://youtu.be/usa796ebEnI) |
+| <img src="https://avatars3.githubusercontent.com/u/370259?s=460&v=4" width="100" /> | Andrew Hill ([@andrewxhill](https://github.com/andrewxhill)) | Textile | SLIDES | [Video](https://youtu.be/uI9eEZQZpfw)
+| <img src="https://avatars2.githubusercontent.com/u/7811297" width="100" /> | Matt Ober ([@obo20](https://github.com/obo20)) | IPFS Industry Trends | SLIDES | [Video](https://youtu.be/LumqUfWd6g0)
+| <img src="https://avatars1.githubusercontent.com/u/687840" width="100" /> | Zhanna Sharipova ([@sharipova](https://github.com/sharipova)) | Anytype | SLIDES | [Video](https://youtu.be/NAF-4AFSqIo)
```
Comment (daviddias) [inline README.md:127]: @sharipova do you happen to have an url with the slides of your Anytype talk?

**Comment by daviddias** [comment]
@sharipova, @obo20, @andrewxhill I'm going to merge this PR as is. When you have the time, can you submit a PR that adds the url to your slides?  Thank you in advance!

File: README.md (hunk 1 above)
Comment (andrewxhill) [reply README.md:125]: just seeing this. yep, they are already committed, https://github.com/ipfs/camp/blob/master/LIGHTNING_TALKS/Andrew_Hill_IPFS_Personal_data_ownership.pdf

Comment (andrewxhill) [reply README.md:125]: oh, looks like you got them, cheers!

Comment (daviddias) [reply README.md:125]: I meant the second talk you did, the one in the last day -> https://www.youtube.com/watch?v=uI9eEZQZpfw

**Pull request merged**